png/apng/fdAT-split-zero-length.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html class="reftest-wait">
+<meta charset="utf-8">
+<title>PNG Third Edition: animated PNG, split fdAT with zero-length chunk</title>
+<link rel="author" title="Chris Lilley" href="mailto:chris@w3.org">
+<link rel="help" href="https://www.w3.org/TR/png-3/#apng-frame-based-animation">
+<link rel="help" href="https://www.w3.org/TR/png-3/#fdAT-chunk">
+<link rel="match" href="apng-lime-rectangle-ref.html">
+<meta name="assert" content="The compressed datastream is then the concatenation of the contents of the data fields of all the fdAT chunks within a frame. ">
+<script>
+    const el = document.querySelector(".reftest-wait");
+    setTimeout(() => {
+        el.classList.remove('reftest-wait');
+        }, "2000");
```

Comment:
Why pass a string rather than a number?

Reply:
Probably because that was what the sample usage I copied from did.
Happy to change to number.